domains/_vercel.tirth.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "tirth",
```

your github username
```suggestion
        "username": "tirth1207",
```